song_scrounger/models/album.py

```diff
@@ -0,0 +1,18 @@
+class Album():
+    def __init__(self, name, spotify_uri, artists, popularity=None):
+        """
+        Params:
+            name (str).
+            spotify_uri (str): e.g. "spotify:track:2ZbTw8awL7EFat9Wz1DIHN".
+            artists ([str]): e.g. ["Don McLean"].
+            popularity (int): 0 <= popularity <= 100, as in Spotify's Web API.
+                None if unspecified.
+        """
+        self.name = name
+        self.spotify_uri = spotify_uri
+        self.artists = artists
```

Comment:
super-nit: you could avoid assigning None to self.popularity by changing the logic a bit... something like

if p is not None and (p is invalid):
  raise ...
self.p = p